samples/model-builder/custom_training_job_sample.py

```diff
@@ -0,0 +1,48 @@
+# Copyright 2021 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+
+from google.cloud import aiplatform
+
+#  [START aiplatform_sdk_custom_training_job_sample]
+def custom_training_job_sample(
+    project,
+    location,
+    bucket,
+    display_name,
+    script_path,
+    script_args,
+    train_image,
+    deploy_image,
+    requirements,
+    replica_count,
+):
+    aiplatform.init(project=project, location=location, staging_bucket=bucket)
+
+    job = aiplatform.CustomTrainingJob(
+        display_name=display_name,
+        script_path=script_path,
+        container_uri=train_image,
```

Comment:
Should the parameter names of the sample be consistent with the parameter names in the MB SDK (in which we try to be consistent with the underlying API)? I can see some value in communicating a different name if we feel like the current parameter name is not descriptive enough for a sample.

Reply:
fixed